### DISCUSSION TITLE
How to use your own Mesh?

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#165] [General]
Hi everyone, I made mesh in ICEM and have generated Polymesh folder by using the command fluentmeshtofoam command in openfoam.
Now I'm unable to understand how to make surfaceMesh.xyz and volumeMesh.xyz to run DAFAOM. Moreover, dafoam has 0.orig folder in the begining. when ./preplProcessing.sh command is run, 1 folder (0) and 2 files( surfaceMesh.xyz and volumeMesh.xyz) are generated.
When using own mesh how to generate these files and how to run dafoam for own mesh?

Kindly guide me.

**Comment by friedenhe** [comment]
Refer to the Rotor37 tutorial on how to convert fluent meshes.

**Reply by usmanshahid90** [reply]
@friedenhe can you please give me the link?
